zubhub_frontend/zubhub/src/components/Navbar/Navbar.jsx

```diff
@@ -62,14 +62,14 @@ const useCommonStyles = makeStyles(commonStyles);
 const anchor = 'left';
 
 /**
- * @function PageWrapper View
+ * @function NavBar View
```

Comment:
This is a Navbar component, in Navbar. jsx

Reply:
It was Pagewrapper.jsx. Guess someone changed it to NavBar.jsx. Be that as it may we shouldn't be making unrelated changes like this especially when the diff is already too large